YouTrackAutomation/Functions/Get-YTProject.ps1

```diff
@@ -61,8 +72,14 @@ function Get-YTProject
     }
 
     $fields = [Uri]::EscapeDataString($fields)
+    $endpoint = "admin/projects?fields=$fields"
+
+    if ($Top)
+    {
+        $endpoint = $endpoint + '&$top=' + "$Top"

```

Comment:
Lint: `$endpoint = "${endpoint}&top=${Top}"`.

Reply:
It needs to be exactly '&$top=' or else it doesn't work correctly. Having that in double quotes will attempt to treat '$top' as a variable.

Reply:
I didn't see the `$` in the query string. Yu can escape it with backtick:

`$endpoint = "${endpoint}&`$top=${Top}"`

Reply:
Ahh the backtick, thank you sir!